### ISSUE TITLE
Move any non-Docker disk creation flows from README to separate docs

### ISSUE THREAD

**Issue by letmaik** [lsds/sgx-lkl#369]
I suggest to remove the following subsections in the "### 2. Creating SGX-LKL disk images with sgx-lkl-disk" section:
- "#### Creating Alpine-based disk images"
- "#### Creating plain disk images"

I also think the following sections can be removed:
- "### 3. Running applications from the Alpine Linux repository"
- "### 4. OpenJDK Java Virtual Machine (JVM)"
- "### 5. Cross-compiling applications for SGX-LKL"

The result is something along the lines of "use docker" and look at the samples for specific workloads (like Java).

EDIT: Instead of removing completely, the non-Docker parts may go into separate docs, with the main README only having a simple Getting Started sample.

**Comment by prp** [comment]
I agree that Java and cross-compilation should go, and instead become sample applications. I still think that there is value in supporting a particular distro (Alpine) directly.

**Comment by letmaik** [comment]
That doesn't scale. Either we offer generic mechanisms (e.g. Docker), or we have to add support for other distributions as well, like `--ubuntu` in the future, but I think that's a bad idea.

**Comment by prp** [comment]
Right, I would only keep it for the moment when Alpine is the only supported distribution. Once that limitations has disappeared, I agree that it should be removed.

**Comment by letmaik** [comment]
I would then (re)move the following:
- "#### Creating plain disk images"
- "### 3. Running applications from the Alpine Linux repository"
- "### 4. OpenJDK Java Virtual Machine (JVM)"
- "### 5. Cross-compiling applications for SGX-LKL"

Note that 3. is outdated and not working anyway, and I think superseded by Docker since that involves a more production type disk setup.
